docs/developer-guide/parallel-execution.md

```diff
@@ -0,0 +1,21 @@
+---
+description: StreamingFast Substreams parallel Execution
```

Comment:
Shouldn't be 
```suggestion
description: StreamingFast Substreams Parallel Execution
```

Reply:
Changed to: StreamingFast Substreams parallel execution, per the style guide. The update will be in the next commit.